tests/test_contracts.py

```diff
@@ -250,6 +250,244 @@ def unary(arg: List[str]) -> None:
         unary(dict)
 
 
+# Testing that postcondition checks regarding function return values pass and fail as expected
+@check_contracts
+def _get_double_valid(num: int) -> int:
+    """
+    Correctly returns twice the number passed as the argument
+
+    Postcondition: $return_value == num * 2
+    """
+    return num * 2
+
+
+@check_contracts
+def _get_double_invalid(num: int) -> int:
+    """
+    Incorrectly returns a number that is not twice the number passed as the argument
+
+    Postcondition: $return_value == num * 2
+    """
+    return (num * 2) + 1
+
+
+def test_get_double_valid() -> None:
+    """Calling the valid implementation of _get_double to get double the number"""
```

Comment:
For all of your test docstrings:

1. Start with the phrase "Test that"
2. End the description with a period.
3. When the description spans more than one line, put the ending `"""` on a newline (not at the end of the previous line). This one really should be enforced by black (the formatter), but I think maybe it's not being run on the test files right now.